### PR TITLE
Carlos practice

### PR DESCRIPTION
**Pull request by rrchcr8**
The files for the examen are in: "practicaenclases" folder

### CODE REVIEW

File: practicaenclases/Practica.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    if num > 0:
+        i = 1
+        Result = 1
+        while i <= num:
+            Result = Result * i
+            i += 1
+        return Result
+
+
+print factorial(0)
```
Comment (sperBruno) [inline practicaenclases/Practica.py]: remove this, you are using unit test to test your method

File: practicaenclases/Practica.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    while j >= 0:
+        chain = chain + num[j]
+        j -= 1
+    if chain == num:
+        return "el numero es capicua"
+    else:
+        return "el numero no es capicua"
+
+
+print is_capicua(55)
```
Comment (sperBruno) [inline practicaenclases/Practica.py]: remove this, you are using unit test to test your method

File: practicaenclases/Practica.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        for j in range(0, n-i-1):
+
+            if arr[j][0] > arr[j+1][0] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
+
+arr=[(3,"Richter"),(2,"Hernando"),(1,"Carlos"),(4,"Rodriguez")]
+
+
+print "Unsorted array is:"
```
Comment (sperBruno) [inline practicaenclases/Practica.py]: remove from line 53 to 60, you are using unit test to test your method

File: practicaenclases/Practica.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    while j >= 0:
+        chain = chain + string[j]
+        j -= 1
+    if chain == string:
+        return "la cadena es palindromo"
+    else:
+        return "la cadena no es palindromo"
+
+
+print is_palindromo("anitalavalatina")
```
Comment (sperBruno) [inline practicaenclases/Practica.py]: remove this, you are using unit test to test your method

File: practicaenclases/Practica.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+
+class AlumnoJALA(Persona):
+    def __init__(self, identificacion, nombre, apellido, codigoAlumno):
+
+        Persona.__init__(self, identificacion, nombre, apellido)
+
+        self.codigoAlumno = codigoAlumno
+
+a = AlumnoJALA("cod 35123456", "carlos", "ruiz", "98765")
```
Comment (sperBruno) [inline practicaenclases/Practica.py]: remove this, you are using unit test to test your method

File: practicaenclases/Practica.py (hunk 1 above)
Comment (rrchcr8) [reply practicaenclases/Practica.py]: commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

File: practicaenclases/Practica.py (hunk 2 above)
Comment (rrchcr8) [reply practicaenclases/Practica.py]: commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

File: practicaenclases/Practica.py (hunk 3 above)
Comment (rrchcr8) [reply practicaenclases/Practica.py]: commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

File: practicaenclases/Practica.py (hunk 2 above)
Comment (rrchcr8) [reply practicaenclases/Practica.py]: commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

File: practicaenclases/Practica.py (hunk 5 above)
Comment (rrchcr8) [reply practicaenclases/Practica.py]: commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

File: practicaenclases/Practica.py (hunk 4 above)
Comment (rrchcr8) [reply practicaenclases/Practica.py]: commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

**Comment by sperBruno** [comment]
merge this PR

**Pull request merged**